NEWS.md

```diff
@@ -11,6 +11,8 @@ next ():
 
 master ():
 
+- plugins: WebOfTrust build0019
```

Comment:
I've invested hours in writing a comprehensive changelog, I would appreciate it very much if you could link it here and in any other announcements surrounding the release:
https://git.io/v5EsM

Reply:
^^^ Ping for the above comment.